<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>538</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>775</width>
    <height>538</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChunSource --轻量级扫描器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ico.ico</normaloff>ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>export</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="currentipRadio">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>当前ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="reslutEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="currentPort">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>port</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>671</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ip</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>当前扫描端口:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentScanLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="TimeOutBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="TimeOutLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>超时时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>clear reslut</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MiniSizeButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resizeApi.png</normaloff>:/resizeApi.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="MaxSizeButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/mini.png</normaloff>:/mini.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/close.png</normaloff>:/close.png</iconset>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>currentPort</zorder>
   <zorder>startButton</zorder>
   <zorder>stopButton</zorder>
   <zorder>exportButton</zorder>
   <zorder>currentipRadio</zorder>
   <zorder>targetEdit</zorder>
   <zorder>reslutEdit</zorder>
   <zorder>portEdit</zorder>
   <zorder>progressBar</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>currentScanLabel</zorder>
   <zorder>TimeOutBox</zorder>
   <zorder>TimeOutLabel</zorder>
   <zorder>clearButton</zorder>
   <zorder>MiniSizeButton</zorder>
   <zorder>MaxSizeButton</zorder>
   <zorder>CloseButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
